tests/raftstore/test_stale_peer.rs

```diff
@@ -152,10 +152,23 @@ fn test_stale_peer_without_data<T: Simulator>(cluster: &mut Cluster<T>) {
 
     pd_client.must_remove_peer(new_region_id, new_peer(3, 4));
 
-    // Wait for max_leader_missing_duration to time out.
-    thread::sleep(cluster.cfg.raft_store.max_leader_missing_duration);
-    // Sleep one more second to make sure there is enough time for the peer to be destroyed.
-    thread::sleep(Duration::from_secs(1));
+    let timer = Instant::now();
```

Comment:
Please modify `test_stale_peer_out_of_region` as this test case to keep them consistent.